tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
 	"compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
Ok. I just didn't see anywhere where imports were actually changed - is it an related change?